modules/deploy/partials/kubernetes/guides/start-streaming.adoc

```diff
@@ -11,18 +11,102 @@ alias rpk-topic="kubectl --namespace <namespace> exec -i -t redpanda-0 -c redpan
 
 . Create a topic called `twitch_chat`:
 +
+[tabs]
+======
+Helm + Operator::
++
+--
+
+.. Create a xref:manage:kubernetes/manage-topics.adoc[Topic resource]:
++
+.`topic.yaml`
+[,yaml]
+----
+apiVersion: cluster.redpanda.com/v1alpha1
+kind: Topic
+metadata:
+  name: twitch_chat
+spec:
+  kafkaApiSpec:
+    brokers:
+      - "redpanda-0.redpanda.<namespace>.svc.cluster.local:9093"
+      - "redpanda-1.redpanda.<namespace>.svc.cluster.local:9093"
+      - "redpanda-2.redpanda.<namespace>.svc.cluster.local:9093"
+    tls:
+      caCertSecretRef:
+        name: "redpanda-default-cert"
+        key: "ca.crt"
+----
+
+.. Apply the Topic resource in the same namespace as your Redpanda cluster:
++
 [,bash]
 ----
-rpk-topic create twitch_chat
+kubectl apply -f topic.yaml --namespace <namespace>
 ----
+
+.. Check the logs of the Redpanda Operator to confirm that the topic was created:
 +
-Example output:
+[,bash]
+----
+kubectl logs -l app.kubernetes.io/name=operator -c manager --namespace <namespace>
+----
++
+You should see that the Redpanda Operator reconciled the Topic resource. For example:
 +
+.Example output
+[%collapsible]
+====
+[,json,.no-copy,lines=5+16]
+----
+{
+  "level":"info",
+  "ts":"2023-09-25T16:20:09.538Z",
+  "logger":"TopicReconciler.Reconcile",
+  "msg":"Starting reconcile loop",
+  "controller":"topic",
+  "controllerGroup":"cluster.redpanda.com",
+  "controllerKind":"Topic",
+  "Topic":{"name":"twitch_chat","namespace":""},
+  "namespace":"",
+  "name":"twitch_chat","reconcileID":"c0cf9abc-a553-48b7-9b6e-2de3cdfb4432"}
+{
+  "level":"info",
+  "ts":"2023-09-25T16:20:09.581Z",
+  "logger":"TopicReconciler.Reconcile",
+  "msg":"reconciliation finished in 43.436125ms, next run in 3s",
+  "controller":"topic",
+  "controllerGroup":"cluster.redpanda.com",
+  "controllerKind":"Topic",
+  "Topic":{"name":"twitch_chat","namespace":""},
+  "namespace":"",
+  "name":"twitch_chat",
+  "reconcileID":"c0cf9abc-a553-48b7-9b6e-2de3cdfb4432",
+  "result":{"Requeue":false,"RequeueAfter":3000000000}
+}
+----
+====
+
+--
+Helm::
++
+--
+
+[,bash]
+----
+rpk-topic create twitch_chat
+----
+
+Example output:
+
 [.no-copy]
 ----
 TOPIC STATUS twitch_chat OK
 ----
 
+--
+======
+
 . Describe the topic:
```

Comment:
NIT:

Using
```
kubectl get topic -o yaml twitch_chat
```
The topic custom resource status should have similar config description.

Reply:
Great suggestion. I need to follow what we did in Redpanda resource. 

https://github.com/redpanda-data/redpanda/blob/dev/src/go/k8s/apis/redpanda/v1alpha1/redpanda_types.go#L109-L110

What columns you would like to see?

Reply:
It would be nice to get the additional info you get from `rpk topic describe`:

```
NAME        twitch-chat
PARTITIONS  1
REPLICAS    1
```

Reply:
I was going to use the Go template to get this detail but couldn't see partitions or replicas anywhere